pkg/rosa/client.go

```diff
@@ -20,16 +22,142 @@ const (
 	ocmAPIURLKey = "ocmApiUrl"
 )
 
+type ocmclient struct {
```

Comment:
OCMClient not ocmclient and please move the OCMClient struct, interface and funcs to another file ex; ocmclient.go